charts/goldpinger/values.yaml

```diff
@@ -143,6 +143,10 @@ serviceMonitor:
   # namespace: monitoring
   interval: 30s
   # honorLabels: true
+  metricRelabelings: {}
```

Comment:
My main concern is the `metricsRelabelings` should be a list and I am not sure I like the part where we are setting it to an empty object `{}` here - I would rather set this to an empty list `[]`? 

(Even though your example below kind of makes it clear we are expecting a list... I still think the type is important here)

Reply:
That's fair. Done.